download_list.json

```diff
@@ -1,8 +1,8 @@
 {
 	"downloads": [
 	  {"file":"ePSXe205.zip", "url":"http://www.epsxe.com/files/ePSXe205.zip"},
-	  {"file":"pcsx2-1.6.0-setup.exe", "url":"http://delivery2.filecroco.com/kits_4/pcsx2-1.6.0-setup.exe"},
-	  {"file":"RetroArch.7z", "url": "https://buildbot.libretro.com/stable/1.8.6/windows/x86_64/RetroArch.7z"},
+	  {"file":"pcsx2-1.6.0-binaries.7z", "url":"https://pcsx2.net/download/releases/windows/send/40-windows/125-pcsx2-1-6-0-binaries.html"},
+	  {"file":"RetroArch.7z", "url": "https://buildbot.libretro.com/stable/1.6.7/windows/x86_64/RetroArch.7z"},
```

Comment:
Is there a reason we'd use 1.6.0 rather than 1.8.6 ?

Reply:
no really, it was just wjat I found in a hurrey, sorry again